src/vaishnaviVJun20_arrayExamples/Armstrong.java

```diff
@@ -0,0 +1,27 @@
+package vaishnaviVJun20_arrayExamples;
+
+//WAP to check if a number is Armstrong or not.
+
+public class Armstrong {
+	
+	void findIfNumberIsArmstrong(int input) {
+		int Originalnumber = input;
+		int sum =0;
+		while(input>0) {
+			int num = input%10;
+			input = input/10;
+			int result = num*num*num;
```

Comment:
**Suggestion:**
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication of same number multiple times (to find square, cube and so on).
i.e. Math.pow((number % 10), 3))

Reply:
--Implemented this logic using this function
--Thanks for the suggestion